pkg/volume/util/util.go

```diff
@@ -612,9 +612,9 @@ func GetLocalPersistentVolumeNodeNames(pv *v1.PersistentVolume) []string {
 // GetPodVolumeNames returns names of volumes that are used in a pod,
 // either as filesystem mount or raw block device, together with list
 // of all SELinux contexts of all containers that use the volumes.
-func GetPodVolumeNames(pod *v1.Pod) (mounts sets.String, devices sets.String, seLinuxContainerContexts map[string][]*v1.SELinuxOptions) {
-	mounts = sets.NewString()
-	devices = sets.NewString()
+func GetPodVolumeNames(pod *v1.Pod) (mounts sets.Set[string], devices sets.Set[string], seLinuxContainerContexts map[string][]*v1.SELinuxOptions) {
+	mounts = sets.New[string]()
+	devices = sets.New[string]()
```

Comment:
Is it better to move this replacement commit into a separate PR?

Reply:
Note to self:

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/124439/pull-kubernetes-linter-hints/1788097670976376832

Reply:
I'm sorry, but it seems that we need to make a fix in this PR because the tests are failing due to the following lint error:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/124439/pull-kubernetes-linter-hints/1788097670976376832